src/main/java/io/pravega/connectors/flink/dynamic/table/FlinkPravegaDynamicTableSource.java

```diff
@@ -114,6 +112,31 @@ public FlinkPravegaDynamicTableSource(DataType producedDataType,
         this.isBounded = isBounded;
     }
 
+    public FlinkPravegaDynamicTableSource(DataType producedDataType,
```

Comment:
This part is redundant, we can remove such constructor

Reply:
Removed